packages/stark-ui/src/modules/dropdown/components/dropdown.component.html

```diff
@@ -1,63 +1,60 @@
-<mat-form-field>
-	<!-- single-selection view -->
-	<mat-select
-		*ngIf="!isMultiSelectEnabled"
-		[id]="dropdownId"
-		[formControl]="formControl"
-		[placeholder]="placeholder | translate"
-		[attr.aria-label]="dropdownName | translate"
-		[required]="required"
-	>
-		<!-- FIXME find a replacement for md-select-header as mat-select-header does not exist yet https://github.com/angular/material2/pull/7835 -->
-		<!--<mat-select-header-->
-		<!--*ngIf="header !== undefined && header !== ''"-->
-		<!--class="stark-dropdown-header"-->
-		<!--translate>-->
-		<!--{{header}}-->
-		<!--</mat-select-header>-->
+<!-- single-selection view -->
+<mat-select
+	*ngIf="!multiSelect"
+	[value]="value"
+	[id]="dropdownId"
+	[placeholder]="placeholder | translate"
```

Comment:
I think we should revisit this... either we translate it ourselves or we ask the developer to pass the translated value using the `translate` pipe... but in my opinion... we should be the ones who translate this value because we already do the same for the options so we should be consistent and translate everything.

I think the same should apply for the date picker... we should translate the placeholder ourselves, otherwise it feels tricky that for some of our components the dev needs to translate the values and for some other he doesn't need to :(